src/app/components/MediaLoader/configs/legacyMedia.ts

```diff
@@ -0,0 +1,49 @@
+import { MediaOverrides } from '#app/models/types/media';
+import filterForBlockType from '#lib/utilities/blockHandlers';
+import {
+  LegacyMediaBlock,
+  ConfigBuilderProps,
+  ConfigBuilderReturnProps,
+} from '../types';
+import buildPlaceholderConfig from '../utils/buildPlaceholderConfig';
+
+export default ({
+  blocks,
+  basePlayerConfig,
+  translations,
+}: ConfigBuilderProps): ConfigBuilderReturnProps => {
+  const legacyMediaBlock: LegacyMediaBlock =
+    filterForBlockType(blocks, 'legacyMedia') ?? {};
+
+  const { model: mediaOverrides }: MediaOverrides =
+    filterForBlockType(blocks, 'mediaOverrides') || {};
+
+  const { image, format, playlist } = legacyMediaBlock?.content ?? {};
+
+  const placeholderConfig = buildPlaceholderConfig({
+    title: mediaOverrides?.pageTitleOverride || '',
+    type: format || 'video',
+    holdingImageURL: image?.href,
+    translations,
+  });
+
+  const mp4Items = playlist
+    ?.filter(item => item.format === 'mp4')
+    ?.map(item => ({ href: item.url, kind: 'programme' }));
```

Comment:
**Optional:** Overkill, but do you want to use destructuring here?

```suggestion
    ?.filter(({ format }) => format === 'mp4')
    ?.map(({ url }) => ({ href:url, kind: 'programme' }));
```

Reply:
I have approved anyway, so don't let this block you

Reply:
Oh sorry, missed these before merging. Good call though, I'll try and remember to do this when working in the MediaLoader again.

Reply:
Oh no worries, it was optional feedback anyways!